scripts/prerelease/generate_prerelease_script.py

```diff
@@ -130,10 +130,10 @@ def main(argv=sys.argv[1:]):
         repositories[repo_name] = source_repo
 
     for repo_name, repo_type, repo_url, version in args.custom_repo:
-        if repo_name in repositories:
-            print("The repository '%s' appears multiple times" % repo_name,
+        if repo_name in repositories and repositories[repo_name]:
+            print("custom_repos option overiding '%s' to '%s': '%s' '%s'. " %
```

Comment:
`overriding 'name' to 'git': 'url' 'version'

"Overriding to git" sounds weird to me.